it/controllers/benefits/medical/EducationalServicesEmploymentBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,427 @@
+/*
```

Comment:
Please run auto-format from Intellij on the file. It will fix spacing issues in the file